crates/viewer/re_renderer/src/renderer/lines.rs

```diff
@@ -762,3 +762,45 @@ impl Renderer for LineRenderer {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{view_builder::TargetConfiguration, Rgba};
+
+    use super::*;
+
+    // Regression test for https://github.com/rerun-io/rerun/issues/8639
+    #[test]
+    fn empty_strips() {
+        re_log::setup_logging();
+        re_log::PanicOnWarnScope::new();
```

Comment:
(off topic) `TestContext` should probably do that too!

Reply:
Yeah, I was thinking opt-out indeed.

Reply:
let's sneak that in here as well, why not